Source/PeterKottas.DotNetCore.WindowsService/ServiceRunner.cs

```diff
@@ -130,36 +142,36 @@ public static int Run(Action<HostConfigurator<SERVICE>> runAction)
                 runAction(hostConfiguration);
                 if (innerConfig.Action == ActionEnum.Run)
                     innerConfig.Service = innerConfig.ServiceFactory(innerConfig.ExtraArguments,
-						new MicroServiceController(() => 
-						{
-							var task = Task.Factory.StartNew(() =>
-							{
-								UsingServiceController(innerConfig, (sc, cfg) => StopService(cfg, sc));
-							});
-						}
-					));
-				else if (innerConfig.Action == ActionEnum.RunInteractive)
-				{
-					var consoleService = new InnerService(innerConfig.Name, () => Start(innerConfig), () => Stop(innerConfig));
-					var consoleHost = new ConsoleServiceHost<SERVICE>(consoleService, innerConfig);
+                        new MicroServiceController(() =>
+                        {
+                            var task = Task.Factory.StartNew(() =>
+                            {
+                                UsingServiceController(innerConfig, (sc, cfg) => StopService(cfg, sc));
+                            });
+                        }
+                    ));
+                else if (innerConfig.Action == ActionEnum.RunInteractive)
+                {
+                    var consoleService = new InnerService(innerConfig.Name, () => Start(innerConfig), () => Stop(innerConfig));
+                    var consoleHost = new ConsoleServiceHost<SERVICE>(consoleService, innerConfig);
 
-					innerConfig.Service = innerConfig.ServiceFactory(innerConfig.ExtraArguments,
-						new MicroServiceController(() =>
-						{
-							var task = Task.Factory.StartNew(() =>
-							{
-								consoleHost.StopService();
-							});
-						}
-					));
+                    innerConfig.Service = innerConfig.ServiceFactory(innerConfig.ExtraArguments,
+                        new MicroServiceController(() =>
+                        {
+                            var task = Task.Factory.StartNew(() =>
+                            {
+                                consoleHost.StopService();
+                            });
+                        }
+                    ));
 
-					// Return the console host run result, so we get some idea what failed if result is not OK
-					return (int)consoleHost.Run();
-				}
+                    // Return the console host run result, so we get some idea what failed if result is not OK
+                    return (int)consoleHost.Run();
+                }
 
-				ConfigureService(innerConfig);
+                ConfigureService(innerConfig);
 
-				return 0;
+                return 0;
```

Comment:
There seems to be a lot of false positive diffs because of formatting changes. Just some extra white spaces I guess. Could you please reformat this doc?